x-pack/plugins/security_solution/cypress/e2e/exceptions/shared_exception_lists_management/manage_exceptions.cy.ts

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  esArchiverLoad,
+  esArchiverUnload,
+  esArchiverResetKibana,
+} from '../../../tasks/es_archiver';
+import { getNewRule } from '../../../objects/rule';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+import { createRule, deleteCustomRule } from '../../../tasks/api_calls/rules';
+import {
+  addExceptionFlyoutItemName,
+  editException,
+  editExceptionFlyoutItemName,
+  submitEditedExceptionItem,
+  submitNewExceptionItem,
+} from '../../../tasks/exceptions';
+import { DETECTIONS_RULE_MANAGEMENT_URL, EXCEPTIONS_URL } from '../../../urls/navigation';
+
+import {
+  CONFIRM_BTN,
+  MANAGE_EXCEPTION_CREATE_BUTTON_MENU,
+  MANAGE_EXCEPTION_CREATE_BUTTON_EXCEPTION,
+  RULE_ACTION_LINK_RULE_SWITCH,
+  EXCEPTION_ITEM_VIEWER_CONTAINER,
+  EXCEPTION_CARD_ITEM_NAME,
+  EXCEPTIONS_LIST_MANAGEMENT_NAME,
+  LINK_TO_SHARED_LIST_RADIO,
+  EXCEPTION_ITEM_HEADER_ACTION_MENU,
+  EXCEPTION_ITEM_OVERFLOW_ACTION_EDIT,
+  EXECPTION_ITEM_CARD_HEADER_TITLE,
+  EXCEPTION_ITEM_OVERFLOW_ACTION_DELETE,
+  EMPTY_EXCEPTIONS_VIEWER,
+} from '../../../screens/exceptions';
+import { goToRuleDetails } from '../../../tasks/alerts_detection_rules';
+import { goToExceptionsTab } from '../../../tasks/rule_details';
+import {
+  createSharedExceptionList,
+  findSharedExceptionListItemsByName,
+  waitForExceptionsTableToBeLoaded,
+} from '../../../tasks/exceptions_table';
+
+describe('Add, edit and delete exception', () => {
+  before(() => {
+    esArchiverResetKibana();
+    esArchiverLoad('exceptions');
+    login();
+    createRule(getNewRule());
+  });
+
+  beforeEach(() => {
+    visitWithoutDateRange(EXCEPTIONS_URL);
+    waitForExceptionsTableToBeLoaded();
+  });
+  after(() => {
+    esArchiverUnload('exceptions');
+  });
+
+  afterEach(() => {
+    deleteCustomRule();
+  });
+
+  const exceptionName = 'My item name';
+  const exceptionNameEdited = 'My item name edited';
+  const FIELD_DIFFERENT_FROM_EXISTING_ITEM_FIELD = 'agent.name';
+  const EXCEPTION_LIST_NAME = 'Newly created list';
+
+  describe('Add, Edit and delete Exception item', () => {
+    it('should create exception item from Shared Exception List page and linked to a Rule', () => {
+      // Click on "Create shared exception list" button on the header
+      cy.get(MANAGE_EXCEPTION_CREATE_BUTTON_MENU).click();
+      // Click on "Create exception item"
+      cy.get(MANAGE_EXCEPTION_CREATE_BUTTON_EXCEPTION).click();
```

Comment:
NIT: Please wrap this into a task method.

Reply:
Done please check [e0e24f7](https://github.com/elastic/kibana/pull/152757/commits/e0e24f779c6479f762916494d5b49264f8886cf6)